app/Http/Controllers/ProjectController.php

```diff
@@ -16,31 +16,31 @@ class ProjectController extends Controller
      */
     public function index(Request $request)
     {
-        if ($request->get('pid')) {
-            $projects = Project::find(16);
-            $projects->members->pluck('id')->toArray();
+      if ($request->get('pid')) {
```

Comment:
Please use 4 spaces for 1 indentation. Here you are only using 2 spaces. I think when you run `vendor/bin/phpcs --standard=prs2 app/` It will tell you (you can use phpcbf to fix must problems automatically `vendor/bin/phpcbf --standard=psr2 app/`